homeassistant/components/tibber/__init__.py

```diff
@@ -48,9 +70,9 @@ async def _close(event):
 
         async def retry_setup(now):
             """Retry setup if a timeout happens on Tibber API."""
-            await async_setup(hass, config, retry_delay=min(2 * retry_delay, 900))
+            await async_setup_entry(hass, entry, retry_delay=min(2 * retry_delay, 900))
```

Comment:
We can just raise ConfigEntryNotReady now instead. The custom retry logic can be removed.